### ISSUE TITLE
Default suggestion for notebook()

### ISSUE THREAD

**Issue by alanedelman** [JuliaLang/IJulia.jl#505]
I'd like to propose that 
notebook()
default to
notebook(dir=".")
rather than $HOME

It seems more intuitive to me, and it gives me the ability
to semicolon cd to my desired directory before typing notebook()

**Comment by stevengj** [comment]
The current behavior was a result of #449 — for people launching `julia` by double-clicking (i.e. on Mac and Windows), it was sometimes starting in an unexpected directory.

**Comment by StefanKarpinski** [comment]
I agree that this behavior is somewhat annoying – I always forget, especially since this used to be the default. Perhaps we should revisit some of the workarounds.

**Comment by stevengj** [comment]
The main problem with defaulting to `dir="."` is on Windows, where double-clicking Julia starts with `pwd() == where julia.exe lives`, which is annoying.  On MacOS, double-clicking Julia stars with `pwd() = homedir`.

**Comment by stevengj** [comment]
(The behavior of the [default working directory for executables in Windows](http://superuser.com/questions/396394/how-do-i-set-an-executables-working-directory-via-the-command-line-prior-to-ex) is jaw-dropping.)
